keras/callbacks.py

```diff
@@ -26,6 +26,7 @@
     import requests
 except ImportError:
     requests = None
+    print("Failed to import requests.")
```

Comment:
This is not necessary. It's ok not to have requests installed (until you try to use the callback that uses it).